tools/provision.sh

```diff
@@ -12,9 +12,23 @@ if [[ $ID == rhel || $ID == centos ]]; then
 fi
 
 sudo mkdir -p /etc/osbuild-composer
-sudo cp -a /usr/share/tests/osbuild-composer/composer/*.toml \
+sudo cp -a /usr/share/tests/osbuild-composer/composer/osbuild-composer.toml \
     /etc/osbuild-composer/
 
+sudo mkdir -p /etc/osbuild-worker
+sudo cp -a /usr/share/tests/osbuild-composer/worker/osbuild-worker.toml \
+    /etc/osbuild-worker/
+
+# if GCP credentials are defined in the ENV, add them to the worker's configuration
+GOOGLE_APPLICATION_CREDENTIALS="${GOOGLE_APPLICATION_CREDENTIALS:-}"
+if [ -n "$GOOGLE_APPLICATION_CREDENTIALS" ]; then
+    # The credentials file must be copied to a different location. Jenkins places
+    # it into /tmp and as a restult, the worker would not see it due to using PrivateTmp=true.
+    GCP_CREDS_WORKER_PATH="/etc/osbuild-worker/$(basename "$GOOGLE_APPLICATION_CREDENTIALS")"
```

Comment:
We might even use a static filename.

Reply:
We can. I probably don't have a strong preference.

Reply:
Me neither, I'm fine with this. :)

Reply:
I changed it :smile: